code/game/objects/effects/decals/posters/voreposters_vr.dm

```diff
@@ -49,4 +49,48 @@
 /datum/poster/vore_13
 	icon_state = "sbsposter13"
 	name = "Ammer"
-	desc = "You see a blue panda on the poster. She's awfully smug about her size."
\ No newline at end of file
+	desc = "You see a blue panda on the poster. She's awfully smug about her size."
+	/datum/poster/vore_14
+	icon_state = "dwposter1"
```

Comment:
```suggestion
	desc = "You see a blue panda on the poster. She's awfully smug about her size."
/datum/poster/vore_14
	icon_state = "dwposter1"
```

You should remove this accidental tab!

Reply:
Another day, another fat finger mistake. Commited.